hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -291,9 +295,28 @@ def test_resource_group_mentioned(self):
 
 class BatchTests(unittest.TestCase):
     def setUp(self):
-        self.backend = ServiceBackend('test')
-        self.gcs_input_dir = 'gs://hail-services/batch-testing/resources'
-        self.gcs_output_dir = os.environ.get('SCRATCH').rstrip('/') + '/output'
+        self.backend = ServiceBackend()
+        bucket_name = get_userinfo()['bucket_name']
+        token = uuid.uuid4()
+        self.gcs_input_dir = f'gs://{bucket_name}/batch-tests/resources'
+        self.gcs_output_dir = f'gs://{bucket_name}/batch-tests/{token}'
+        gcs_client = google.cloud.storage.Client(
+            project='hail-vdc',
+            credentials=google.oauth2.service_account.Credentials.from_service_account_file(
+                '/test-gsa-key/key.json'))
+        bucket = gcs_client.bucket(bucket_name)
+        if not bucket.blob('batch-tests/resources/hello (foo) spaces.txt').exists():
+            print('uploading 512 MB to GCS, this may take some time, but only '
+                  'happens once per-user')
+            bucket.blob('batch-tests/resources/random_file_512mb').upload_from_string(
+                os.urandom(512 * 1024 * 1024))
```

Comment:
I think this is going to break the test in my gsutil replacement branch. I think I can remove the 512 MB test since I replaced it with a smaller 16KB with 4 way parallelism with block size of 4KB.

Reply:
I think we should just get rid of this random file for now.